ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -73,9 +76,10 @@
       <p>{{dc}}</p>
     {{#each upstreams as |item|}}
       <TopologyMetrics::Card
+        @nspace={{@nspace}}
+        @dc={{@dc}}
```

Comment:
Not entirely sure if these should use the dc and nspace from the upstream, I think we spoke about this and I'm guessing its not the case as yet.